rel/server/env.sh.eex

```diff
@@ -1,26 +1,32 @@
-NODE_DEFAULT="livebook_server"
-
 if [ "$LIVEBOOK_CLUSTER" = "fly" ]; then
   export ERL_AFLAGS="-proto_dist inet6_tcp"
   export LIVEBOOK_CLUSTER="dns:${FLY_APP_NAME}.internal"
-  NODE_DEFAULT="${FLY_APP_NAME}-${FLY_IMAGE_REF##*-}@${FLY_PRIVATE_IP}"
+  if [ ! -z "${LIVEBOOK_NODE}" ];
+    then export LIVEBOOK_NODE="${FLY_APP_NAME}-${FLY_IMAGE_REF##*-}@${FLY_PRIVATE_IP}";
+  fi
 fi
 
 if [ -f "${RELEASE_ROOT}/user/env.sh" ]; then
   . "${RELEASE_ROOT}/user/env.sh"
 fi
 
-export RELEASE_MODE=interactive
-export RELEASE_NODE=${LIVEBOOK_NODE:-${RELEASE_NODE:-${NODE_DEFAULT}}}
-export RELEASE_DISTRIBUTION="name"
-
 if [ "$LIVEBOOK_EPMDLESS" = "true" ] || [ "$LIVEBOOK_EPMDLESS" = "1" ]; then
   export ELIXIR_ERL_OPTIONS="${ELIXIR_ERL_OPTIONS} -epmd_module Elixir.Livebook.EPMD -start_epmd false -erl_epmd_port 0"
 fi
 
+export RELEASE_MODE="interactive"
+export RELEASE_DISTRIBUTION="none"
+
+# Mirror these values, so that it is easier to use "bin/release rpc",
+# though it still requires setting RELEASE_DISTRIBUTION=name
+if [ ! -z "${LIVEBOOK_NODE}" ]; then export RELEASE_NODE=${LIVEBOOK_NODE}; fi
 if [ ! -z "${LIVEBOOK_COOKIE}" ]; then export RELEASE_COOKIE=${LIVEBOOK_COOKIE}; fi
-cookie_path="${RELEASE_ROOT}/releases/COOKIE"
-if [ ! -f $cookie_path ] && [ -z "$RELEASE_COOKIE" ]; then
-  RELEASE_COOKIE=$(cat /dev/urandom | env LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
-  echo "$RELEASE_COOKIE" > "$cookie_path" || export RELEASE_COOKIE
```

Comment:
FTR we used to attempt to write the random cookie to a file and respect `RELEASE_COOKIE`, such that the cookie was the same on subsequent boots. This was necessary because in the past Livebook Desktop was calling [`bin/release rpc`](https://github.com/livebook-dev/livebook/pull/1032/files#diff-5688561ea5a829b9c130fe83416fdde6b022f90d5dfbe55f1c430a3a1144acd3R149-R157) and so it had to use the same cookie. We no longer do that though, so I simplified the code to no longer write the file.